labelbox/schema/ontology.py

```diff
@@ -159,29 +159,25 @@ def from_dict(cls, dictionary: Dict[str, Any]) -> Dict[str, Any]:
                    feature_schema_id=dictionary.get("featureSchemaId", None),
                    scope=cls.Scope(dictionary.get("scope", cls.Scope.GLOBAL)))
 
-    def asdict(self) -> Dict[str, Any]:
+    def asdict(self, is_subclass: bool = False) -> Dict[str, Any]:
         if self.class_type in self._REQUIRES_OPTIONS \
                 and len(self.options) < 1:
             raise InconsistentOntologyException(
                 f"Classification '{self.instructions}' requires options.")
-        return {
-            "type":
-                self.class_type.value,
-            "instructions":
-                self.instructions,
-            "name":
-                self.name,
-            "required":
-                self.required,
+        classification = {
+            "type": self.class_type.value,
+            "instructions": self.instructions,
+            "name": self.name,
+            "required": self.required,
             "options": [o.asdict() for o in self.options],
-            "schemaNodeId":
-                self.schema_id,
-            "featureSchemaId":
-                self.feature_schema_id,
-            "scope":
-                self.scope.value
-                if self.scope is not None else self.Scope.GLOBAL.value
+            "schemaNodeId": self.schema_id,
+            "featureSchemaId": self.feature_schema_id
         }
+        if is_subclass:
+            return classification
+        classification[
+            "scope"] = self.scope.value if self.scope is not None else self.Scope.GLOBAL.value
```

Comment:
Instead of "not None" could you just say `if self.scope`? (I'm not a python expert so not 100% sure)

Reply:
I think so! It may be a minor nit though but I can change if it is more clear